src/modal/modal-backdrop.ts

```diff
@@ -3,7 +3,8 @@ import {Component, Input} from '@angular/core';
 @Component({
   selector: 'ngb-modal-backdrop',
   template: '',
-  host: {'[class]': '"modal-backdrop fade show" + (backdropClass ? " " + backdropClass : "")'}
+  host:
+      {'[class]': '"modal-backdrop fade show" + (backdropClass ? " " + backdropClass : "")', 'style': 'z-index: 1050'}
```

Comment:
I'm surprised that we don't need to adjust `z-index` on windows themselves...

Reply:
@pkozlowski-opensource it's ok to not adjust manually `z-index`.
According to [W3C](
http://www.w3.org/TR/CSS2/visuren.html#propdef-z-index):

> Each box belongs to one stacking context. Each positioned box in a given stacking context has an integer stack level, which is its position on the z-axis relative other stack levels within the same stacking context. Boxes with greater stack levels are always formatted in front of boxes with lower stack levels. Boxes may have negative stack levels. **Boxes with the same stack level in a stacking context are stacked back-to-front according to document tree order.**

So it should be fine for us.